pytest.ini

```diff
@@ -11,6 +11,7 @@ markers =
     order: Configure test order
     early: Run fixtures early
     redhat_internal_dependency: Tests which have a dependency on an RedHat internal resource
+    x86_default_setup: Run on standard cluster configuration
```

Comment:
this marker is designed to run all tests that works with global_config.py? (default one?)

Reply:
yes, as mentined in https://github.com/RedHatQE/openshift-virtualization-tests/pull/1028/files#r2099628906

renamed marker